### PR TITLE
use ember-auto-import instead of ember-browserify

### PR DESCRIPTION
**Pull request by BryanCrotaz**
`ember-browserify` is deprecated. Use `ember-auto-import` instead.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,12 +24,13 @@
   },
   "dependencies": {
     "broccoli-persistent-filter": "^1.4.3",
+    "ember-auto-import": "^1.2.2",
     "ember-cli-babel": "^6.6.0",
+    "mocha": "^4.0.1",
```
[the remark this answers is not in the document]
Comment (simonihmig) [reply package.json]: Agree, seems wrong!

File: blueprints/mocha/ember-cli-yadda/files/tests/helpers/yadda.js [hunk 2]
```diff
@@ -1,2 +1,2 @@
-import yadda from 'npm:yadda';
+import yadda from 'yadda';
```
Comment (simonihmig) [inline blueprints/mocha/ember-cli-yadda/files/tests/helpers/yadda.js]: As these files are part of the blueprint, they won't get updated automatically in a user's app. What would happen if `npm:` is unchanged? Would that break things, i.e. the changes here are breaking changes that require a major version bump?

And what's actually the point of having this file still around, as it is just reexporting the unchanged import?

**Comment by sfbwalder** [comment]
I was testing this pr locally and wasn't sure what the right approach to exposing yadda was.  Here is a great explanation and choices:  https://github.com/ef4/ember-auto-import/issues/84

**Comment by sfbwalder** [comment]
Closed in favor of #79

**Pull request closed without merging**